css/site.styl

```diff
@@ -21,6 +21,7 @@ normalize()
 
 // TYPOGRAPHY
 
+@import 'fonts'
```

Comment:
i dont rly see the point of the fonts file if there is a type file too. could just put it there